### PR TITLE
Very selectively fix some PEP 8 issues and other manual formatting changes

### PR DESCRIPTION
**Pull request by dylanmtaylor**

### CODE REVIEW

File: profiles/cinnamon.py [hunk 1]
```diff
@@ -4,7 +4,17 @@
...
+	"cinnamon",
+	"system-config-printer",
+	"gnome-keyring",
+	"gnome-terminal",
+	"blueberry",
+	"metacity",
+	"lightdm",
+	"lightdm-gtk-greeter",
+]
+
```
Comment (Torxed) [inline profiles/cinnamon.py:17]: I really like this one! Way cleaner!

**Review by Torxed — APPROVED** [review]
So far these changes all look ok.

File: archinstall/lib/installer.py [hunk 2]
```diff
@@ -320,18 +316,18 @@ def minimal_installation(self):
 				if 'encrypt' not in self.HOOKS:
 					self.HOOKS.insert(self.HOOKS.index('filesystems'), 'encrypt')
 
-		if not(hasUEFI()):
+		if not (hasUEFI()):
```
Comment (Torxed) [inline archinstall/lib/installer.py:319]: We could remove the excessive `()` around `hasUEFI()` as well.

File: archinstall/lib/user_interaction.py [hunk 3]
```diff
@@ -259,16 +276,16 @@ def get_keyboard_input(self, strip_rowbreaks=True, end='\n'):
 
 		poller.register(sys.stdin.fileno(), select.EPOLLIN)
 
-		EOF = False
-		while EOF is False:
+		eof = False
+		while eof is False:
```
Comment (Torxed) [inline archinstall/lib/user_interaction.py:280]: `EOF` is a known term, more accurately it's an abbreviation and those are spelled out in capital letters AFAIK.

File: examples/guided.py [hunk 4]
```diff
@@ -1,20 +1,23 @@
...
 from archinstall.lib.hardware import hasUEFI
-from archinstall.lib.profiles import Profile
 
 if archinstall.arguments.get('help'):
 	print("See `man archinstall` for help.")
 	exit(0)
 
 # For support reasons, we'll log the disk layout pre installation to match against post-installation layout
-archinstall.log(f"Disk states before installing: {archinstall.disk_layouts()}", level=archinstall.LOG_LEVELS.Debug)
+archinstall.log(f"Disk states before installing: {archinstall.disk_layouts()}", level=archinstall.LogLevels.Debug)
```
Comment (Torxed) [inline examples/guided.py:13]: Note to self, this should be `logging.DEBUG` not `LogLevels.Debug`.
It's an issue outside of this PR but caught it here. `LOG_LEVELS` is deprecated and going away.

File: examples/guided.py [hunk 5]
```diff
@@ -319,7 +319,7 @@ def perform_installation(mountpoint):
 			archinstall.use_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors for the live medium
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
-			if archinstall.arguments['mirror-region'].get("mirrors",{})!= None:
+			if archinstall.arguments['mirror-region'].get("mirrors", {}) is not None:
```
Comment (Torxed) [inline examples/guided.py:322]: Issue outside of this PR:
But I believe this check is incorrect. `{}` will trigger as `True` against `is not None` and carry on attempting to set an empty mirror list? Which is not what we want. Perhaps omit anything after and just do `if get("mirrors", {})` to make sure it evaluates `True` only if mirrors were actually given? Or do `.get("mirrors", None) is not None`.

File: examples/guided.py [hunk 6]
```diff
@@ -387,7 +387,8 @@ def perform_installation(mountpoint):
 				pass
 
 	# For support reasons, we'll log the disk layout post installation (crash or no crash)
-	archinstall.log(f"Disk states after installing: {archinstall.disk_layouts()}", level=archinstall.LOG_LEVELS.Debug)
+	archinstall.log(f"Disk states after installing: {archinstall.disk_layouts()}", level=archinstall.LogLevels.Debug)
```
Comment (Torxed) [inline examples/guided.py:390]: Again, `logging.DEBUG` as `LOG_LEVELS` (or `LogLevels`) is deprecated.

**Review by Torxed — CHANGES_REQUESTED** [review]
Some minor changes that should but also could be made before this goes in.

File: archinstall/lib/user_interaction.py (hunk 3 above)
Comment (dylanmtaylor) [reply archinstall/lib/user_interaction.py:280]: variables in local functions are supposed to be lowercase

File: archinstall/lib/installer.py [hunk 7]
```diff
@@ -268,20 +265,21 @@ def post_install_enable_iwd_service(*args, **kwargs):
...
-					
 
 		return True
 
 	def detect_encryption(self, partition):
 		if partition.encrypted:
 			return partition
-		elif partition.parent not in partition.path and Partition(partition.parent, None, autodetect_filesystem=True).filesystem == 'crypto_LUKS':
+		elif partition.parent not in partition.path and Partition(partition.parent, None,
+																  autodetect_filesystem=True).filesystem == 'crypto_LUKS':
```
[the remark this answers is not in the document]
Comment (dylanmtaylor) [reply archinstall/lib/installer.py]: Resolved in latest commit

File: examples/guided.py (hunk 4 above)
Comment (dylanmtaylor) [reply examples/guided.py]: Resolved in latest commit

File: archinstall/lib/installer.py (hunk 2 above)
Comment (dylanmtaylor) [reply archinstall/lib/installer.py]: Resolved in latest commit

File: archinstall/lib/disk.py [hunk 8]
```diff
@@ -216,12 +216,13 @@ def uuid(self) -> Optional[str]:
...
 			return partition.get('partuuid', None)
 		return None
+
 	@property
 	def encrypted(self):
 		return self._encrypted
 
 	@encrypted.setter
-	def encrypted(self, value :bool):
+	def encrypted(self, value: bool):
```
[the remark this answers is not in the document]
Comment (dylanmtaylor) [reply archinstall/lib/disk.py:225]: I actually like it much better the "standard way". Having a space before the colon feels awkward and just doesn't look right to me. The way it is in this change makes it very clear that value is of type bool.

File: examples/guided.py (hunk 5 above)
Comment (dylanmtaylor) [reply examples/guided.py]: Changed to your suggestion.

**Pull request merged**